easybuild/easyblocks/generic/configuremake.py

```diff
@@ -184,6 +184,7 @@ def extra_options(extra_vars=None):
                                  " False implies to leave it up to the configure script)", CUSTOM],
             'configure_cmd': [DEFAULT_CONFIGURE_CMD, "Configure command to use", CUSTOM],
             'configure_cmd_prefix': ['', "Prefix to be glued before ./configure", CUSTOM],
+            'configure_no_prefix': [False, "Avoid adding a prefix to the configure command", CUSTOM],
```

Comment:
The different uses of `prefix` in `extra_vars` is causing me confusion. I.e. this has no impact on `configure_cmd_prefix`. I think the following is clearer:

```
"Avoid passing an install directory (such as through --prefix) to the configure command"
```

Also if someone sets `configure_no_prefix = True` and specifies a `prefix_opt` we should warn or exit.

Reply:
And if `configure_no_prefix` would be better as `configure_without_installdir`.

Reply:
that said, it's still sensible. changes made in latest commit